int16 bidder_id
int16 job_id